swap/swap_test.go

```diff
@@ -1083,3 +1084,29 @@ func (d *dummyMsgRW) ReadMsg() (p2p.Msg, error) {
 func (d *dummyMsgRW) WriteMsg(msg p2p.Msg) error {
 	return nil
 }
+
+func TestFileLogger(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping test in short mode.")
+	}
+
+	handler, err := log.FileHandler("/home/git/go/src/github.com/ethersphere/swarm/testlog/test.log", log.TerminalFormat(true))
+	if err != nil {
+		t.Fatal(err)
+	}
+	log.Root().SetHandler(log.LvlFilterHandler(log.Lvl(*loglevel), handler))
+	log.Error("Test")
+}
+
+func TestRotatingFileLogger(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping test in short mode.")
+	}
+
+	handler, err := log.RotatingFileHandler("/home/git/go/src/github.com/ethersphere/swarm/testlog/testRotating.log", 200, log.TerminalFormat(true))
```

Comment:
Cleanup?

Reply:
Same as the other test, I thought about.
But first I wanted some feedback on the implementation before developing all the test.
It definitely needs cleanup, I agreed on that.